WalletWasabi.Daemon/Global.cs

```diff
@@ -406,6 +406,8 @@ public async Task DisposeAsync()
 				UpdateManager.Dispose();
 				Logger.LogInfo($"{nameof(UpdateManager)} is stopped.");
 
+				CoinPrison.ToFile();
```

Comment:
I think that `CoinPrison` should inherit `IDisposable` and this `ToFile` moved to the `Dispose` function to avoid making it `public` and for consistency

Reply:
Makes sense, done.